ui/packages/form/src/Range.svelte

```diff
@@ -24,6 +24,7 @@
 
 	$: dispatch("change", value);
 	const clamp = () => {
+		dispatch("release", value);
```

Comment:
Dispatching the release event here so that it is triggered when users finish typing and focus out of the number input. Otherwise app developers would have to specify both change and release event handlers which would defeat the purpose of the release event.